packages/ingest/parse-pdr.js

```diff
@@ -175,43 +143,4 @@ async function granuleFromFileGroup(fileGroup, pdrName, collectionConfigStore) {
     granuleSize: files.reduce((total, file) => total + file.size, 0)
   };
 }
-
-/**
- * Parse a PDR file
- *
- * @param {string} pdrFilePath - the file to be parsed
- * @param {Object} collectionConfigStore - a CollectionConfigStore object used
- *   for fetching collection configurations
- * @param {string} pdrName - the name of the PDR
- * @returns {Promise<Object>} an object representing the PDR
- */
-exports.parsePdr = async function parsePdr(pdrFilePath, collectionConfigStore, pdrName) {
```

Comment:
Yeah, I don't see this used anywhere. It seems like an early form of the `parse-pdr` task handler